test/test_lang_bind_helper.cpp

```diff
@@ -10459,6 +10459,104 @@ TEST(LangBindHelper_HandoverTableViewWithLinkView)
     }
 }
 
+TEST(Query_ListOfPrimitivesHandover)
+{
+    SHARED_GROUP_TEST_PATH(path);
+    std::unique_ptr<Replication> hist(make_in_realm_history(path));
+    SharedGroup sg(*hist);
+    auto& group = sg.begin_read();
+
+    std::unique_ptr<Replication> hist_w(make_in_realm_history(path));
+    SharedGroup sg_w(*hist_w, SharedGroupOptions(crypt_key()));
+    Group& group_w = const_cast<Group&>(sg_w.begin_read());
+    SharedGroup::VersionID vid;
+
+    std::unique_ptr<SharedGroup::Handover<TableView>> handover;
+    {
+        LangBindHelper::promote_to_write(sg_w);
+
+        TableRef t = group_w.add_table("table");
+        DescriptorRef subdesc;
+        size_t int_col = t->add_column(type_Table, "integers", false, &subdesc);
+        subdesc->add_column(type_Int, "list", nullptr, true);
+
+        t->add_empty_row(10);
+
+        auto set_list = [](TableRef subtable, const std::vector<int64_t>& value_list) {
+            size_t sz = value_list.size();
+            subtable->clear();
+            subtable->add_empty_row(sz);
+            for (size_t i = 0; i < sz; i++) {
+                subtable->set_int(0, i, value_list[i]);
+            }
+        };
+
+        set_list(t->get_subtable(int_col, 0), std::vector<int64_t>({1, 2, 3}));
+        set_list(t->get_subtable(int_col, 1), std::vector<int64_t>({1, 3, 5, 7}));
+        set_list(t->get_subtable(int_col, 2), std::vector<int64_t>({100, 400, 200, 500, 300}));
+
+        auto q = t->get_subtable(int_col, 2)->column<Int>(0) > 225;
```

Comment:
if you change this to `get_subtable(int_col, 3)` then `import_from_handover()` crashes. Note that subtable 3 is a "degenerate" subtable, i.e., in the .realm database file, there is no table object stored, and the parent table just contains a 0 ref... this might need to be special handled?